### DISCUSSION TITLE
New Platform Release v3.1.0 is now available

### DISCUSSION THREAD

**Discussion by sujen1412** [MAAP-Project/Community#790] [Announcements]
### Notable updates in this release:

* Disabled jupyter collaboration feature to address the issue with notebook resetting periodically and with blank notebooks.
* Mitigated the “che-in-che” issue for most users running concurrent workspaces.
* Improved Capacity by 3x for new OPS

Detailed list of updates in this release: 
https://github.com/MAAP-Project/Community/milestone/19?closed=1

### Known Issues:
There are some rare cases when the “che-in-che” issue might show up, please contact the devs if you experience this. 

### How to get the new release:
Currently running workspaces will not automatically update to the new release. 
Users are requested to start up a new workspace with version 3.1.0

Release notes on RtD stable URL: https://docs.maap-project.org/en/latest/release_notes.html

**Comment by rtapella** [comment]
Updated the release notes link.

**Comment by rtapella** [comment]
Does anyone have opinions on the presentation of info on Read-The-Docs
https://docs.maap-project.org/en/latest/release_notes.html

vs. Github
https://github.com/MAAP-Project/Community/milestone/19?closed=1

Any thoughts on the messaging of what's included in a release will help us improve the release notes in the future.
